server/node/src/main/java/org/kaaproject/kaa/server/admin/services/messaging/MessagingServiceImpl.java

```diff
@@ -202,12 +206,29 @@ public Void call() throws Exception {
     }
   }
 
+  /*
+  * Use instead of UrlParams#generateParamsUrl() cause it uses com.google.gwt.http.client.URL that stop
+  * execution of code on server side.
+  * */
+  private String generateParamsUrl(Map<String, String> paramsMap) {
+    String paramsUrl = "";
+    for (String key : paramsMap.keySet()) {
+      String val = paramsMap.get(key);
+      if (paramsUrl.length() > 0) {
+        paramsUrl += "&";
+      }
+      paramsUrl += key + "=" + UrlEscapers.urlPathSegmentEscaper().escape(val);
+    }
+    return paramsUrl;
+  }
+
+
   /**
    * Send password after reset.
    *
-   * @param username  the username
-   * @param password  the password
-   * @param email     the email
+   * @param username the username
+   * @param password the password
+   * @param email    the email
```

Comment:
Could you improve the doc or remove it altogether. (It is stupid and just wastes developers' time.)